text/0011-storage-token.md

```diff
@@ -16,20 +16,34 @@ The most straightforward way of pricing blockchain resources is to charge a fixe
 
 To remedy this issue, Ethereum overprices storage, which curbs the growth rate of the global state. Ethereum also implements a *gas refund* mechanism, where users can discount gas usage of their transactions, if they free up previously used space. However, this arrangement does not ensure an upper bound to global state size. It grows at a linear rate, and one can see that it will require full nodes to have higher and more expensive storage[^1]. Unless an alternative model is implemented, the requirements could increase indefinitely, decreasing the number of full nodes due to increased costs. This can have adverse affects on the network.
 
-Here, we propose an alternative model which leverages the scarcity of CASPER to limit the growth of global state, and increase the allocative efficiency of storage.
+Here, we propose an alternative model which leverages the scarcity of CSPR to limit the growth of global state, and increase the allocative efficiency of storage.
 
 ## Guide-level explanation
 
 [guide-level-explanation]: #guide-level-explanation
 
-We introduce the concept of *data ownership*: every bit of data stored in the global state is associated with an account. Accounts must hold storage tokens in order to store data, denoted internally by the symbol STR. STR is indivisible and is the smallest denomination of itself. 1 STR gives its holder the right to store 1 byte of data.
+Every bit of data stored in the global state is associated with an account. Accounts must hold storage tokens in order to store data, denoted internally by the symbol STR. STR is indivisible and is the smallest denomination of itself. 1 STR gives its holder the right to store 1 byte of data.
```

Comment:
`bit` → `byte`
But still: It doesn't need to be associated with an account for the purposes of this proposal, does it? I pay when I store it, and you get back the STR when you free the data.

Reply:
I updated the sentence again
> Every byte of data stored in the global state belongs to an account.
Does it sound OK now?

Reply:
I don't understand why the association between the byte and the account is relevant for this proposal.

Reply:
I say to highlight the fact that there can be no data that is stored by the users without going through the STR minting process. If it still sounds irrelevant, I'll remove the sentence.

Reply:
Sure, it's probably fine to leav it like that.
It just reads a bit as if that association were still relevant after the initial store operation.

Reply:
Removed that first sentence and added the point to the next paragraph. This should be resolved now.